piksi_multi_rtk_ros/cfg/piksi_multi_driver_settings.yaml

```diff
@@ -1,20 +1,56 @@
-# General Settings.
-serial_port: '/dev/ttyUSB0'
-baud_rate: 230400
+# General Settings
+
+# interface: specifies the interface over which to connect.
+# supported values are 'serial' and 'tcp'
+# if this value is not valid, the driver will default to serial communication
+interface: 'serial'
+
 base_station_mode: false
-debug_mode: false # if true, output every topic read by this driver
+
+# debug_mode: if true, output every topic read by this driver
 # whenever possible, use network broadcast, not global ip broadcast
```

Comment:
The comment " whenever possible, use ..." geos before to parameter "broadcast_addr" and not before param "debug_mode".

Reply:
good catch. will update.